include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1146,6 +1130,26 @@ struct OrtApi {
   */
   ORT_API2_STATUS(ModelMetadataGetGraphDescription, _In_ const OrtModelMetadata* model_metadata,
                   _Inout_ OrtAllocator* allocator, _Outptr_ char** value);
+
+  /**
+  * Use this API to create the configuration of a CUDA Execution Provider
+  */
+  ORT_API2_STATUS(CreateCUDAProviderOptions, _Outptr_ OrtCUDAProviderOptions** out);
+
+  /**
+  * Use this API to set the appropriate configuration knobs of a CUDA Execution Provider
+  * Please refer to the following on different key/value pairs to configure a CUDA EP and their meaning:
```

Comment:
>Please refer to the following on [](start = 4, length = 32)

Sure, but we need to specifically state the format and the content for the params. Strings are UTF8, do they contain zero terminators, etc.